pkg/karmadactl/get.go

```diff
@@ -0,0 +1,559 @@
+package karmadactl
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	metav1beta1 "k8s.io/apimachinery/pkg/apis/meta/v1beta1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	sets2 "k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/cli-runtime/pkg/printers"
+	"k8s.io/cli-runtime/pkg/resource"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/kubectl/pkg/cmd/get"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/scheme"
+	"k8s.io/kubectl/pkg/util/i18n"
+
+	"github.com/karmada-io/karmada/pkg/karmadactl/options"
+)
+
+const (
+	rbLabelNSKey = "propagationpolicy.karmada.io/namespace"
+)
+
+var (
+	getIn  = os.Stdin
+	getOut = os.Stdout
+	getErr = os.Stderr
+
+	podColumns = []metav1.TableColumnDefinition{
+		{Name: "Cluster", Type: "string", Format: "", Priority: 0},
+		{Name: "ADOPTION", Type: "string", Format: "", Priority: 0},
```

Comment:
What does `ADOPTION` mean?

Reply:
Is this workload or svc or other resource managed by karmada cluster. Y is Yes, N is No.